docs/admin_api/rooms.md

```diff
@@ -279,6 +281,34 @@ Response:
 Once the `next_token` parameter is no longer present, we know we've reached the
 end of the list.
 
+# Create Room API
+
+The Create Room admin API allows server admins to create a new room on the server.
+It is possible to specify an owner for the room other than the requester himself.
+In that case, the server admin, who made the request **does not** become a member
+of the created room.
+
+## Usage
+
+The API is:
+
+```json
+POST /_synapse/admin/v1/rooms
+```
+
+the body parameters are the same as in
+<https://matrix.org/docs/spec/client_server/r0.6.1#post-matrix-client-r0-createroom>
+with one additional optional parameter
+```json
+{
+    "owner": "@someuser:example.com"
```

Comment:
I think we should avoid referring to this as "owner" - the creator is not necessarily the "owner" of the room.

```suggestion
    "creator": "@someuser:example.com"
```

Reply:
In the current implementation, the admin user performing the request would still be the `creator` of the room, but will not become a member. Instead he announces another user as "owner" or single member with room admin rights.

Reply:
OK, understood. Then I will accept your renames.